### PR TITLE
Context API rdocs

### PR DESCRIPTION
**Pull request by josh**
Kicking off some initial `Duktape::Context` rdocs.

screen shot 2015-03-18 at 7 49 57 pm

### CODE REVIEW

**Comment by josh** [comment]
@judofyr how is this looking for a start on inline docs? I'll probably open a separate PR for improving the README.

File: ext/duktape/duktape_ext.c [hunk 1]
```diff
@@ -431,6 +452,16 @@ static void ctx_get_nested_prop(struct state *state, VALUE props)
...
 }
 
+/*
+ * call-seq:
+ *   get_prop(name) -> obj
+ *
+ * Access the property of the global object.
+ *
+ *     ctx.exec_string("var n = 42", "foo.js")
+ *     ctx.get_prop("n") #=> 42
```
Comment (judofyr) [inline ext/duktape/duktape_ext.c]: `get_prop` supports nested access as well, right?

Comment (josh) [inline ext/duktape/duktape_ext.c]: Ah, yeah, missed that.

Comment (josh) [inline ext/duktape/duktape_ext.c]: Fixed.

**Comment by judofyr** [comment]
Looks good.

**Pull request merged**